python/tests/test_autograd.py

```diff
@@ -3,6 +3,7 @@
 import unittest
 
 import mlx.core as mx
+
```

Comment:
Just random nit: any reason you edited all of these python test files?

Reply:
some formatter did that, I'll see if I can revert and it'll leave them alone (might have run it before pulling the hooks or something? not sure)

Reply:
Yea that's possible. It's minor so don't worry if it's tedious. As long as it passes the pre-commit hooks it's ok. You can try running `pre-commit run --all-files` to see if it yells at you. (Our CI will run that as well once we run the tests for this)

Reply:
this should be good in the latest commit